admin/admin.html

```diff
@@ -0,0 +1,208 @@
+<!doctype html>
```

Comment:
That's probably not a bad idea. I like the idea of being able to add/remove buttons from backend and frontend in a single commit though. Maybe later?

Reply:
Already regretting that 'sanction' button, would like to see if it's useful before removing it though

Reply:
I think that in the future it will be better to bring such functionality (administration) into a separate project, such as a master-server, which will manage several game servers at once.